### PR TITLE
added refund notifications job

### PR DESCRIPTION
**Pull request by larslunde-hmcts**
…icies to img automation

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by reespozzi — COMMENTED** [review]
Rest looks OK besides image-repo comment

**Review by reespozzi — DISMISSED** [review]

File: k8s/namespaces/fees-pay/refund-notifications-job/refund-notifications-job.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/payment-jobs
+  values:
+    job:
+      image: "hmctspublic.azurecr.io/payment/jobs:prod-0a0ded6-20210716134918"   #{"$imagepolicy": "flux-system:refund-notifications-job"}
```
Comment (thomast1906) [inline k8s/namespaces/fees-pay/refund-notifications-job/refund-notifications-job.yaml:17]: Just a comment, can you sort spacing?

File: k8s/namespaces/fees-pay/refund-notifications-job/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: refund-notifications-job
+  namespace: fees-pay
+spec:
+  releaseName: refund-notifications-job
```
Comment (thomast1906) [inline k8s/namespaces/fees-pay/refund-notifications-job/demo.yaml:7]: Can you remove duplication` releaseName: refund-notifications-job`

File: apps/fees-pay/refund-notifications-job/image-policy.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImagePolicy
+metadata:
+  name: refund-notifications-app
```
Comment (thomast1906) [inline apps/fees-pay/refund-notifications-job/image-policy.yaml:4]: Is this correct ? `refund-notifications-app`

Comment (larslunde-hmcts) [reply apps/fees-pay/refund-notifications-job/image-policy.yaml:4]: It has been corrected to 'refund-notifications-job'

File: k8s/namespaces/fees-pay/refund-notifications-job/demo.yaml (hunk 2 above)
Comment (larslunde-hmcts) [reply k8s/namespaces/fees-pay/refund-notifications-job/demo.yaml:7]: done

File: k8s/namespaces/fees-pay/refund-notifications-job/demo.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  values:
+    job:
+      image: "hmctspublic.azurecr.io/payment/jobs:pr-32-0a0ded6-20210716134918"   #{"$imagepolicy": "flux-system:demo-refund-notifications-job"}
```
[the remark this answers is not in the document]
Comment (larslunde-hmcts) [reply k8s/namespaces/fees-pay/refund-notifications-job/demo.yaml:10]: done

File: k8s/namespaces/fees-pay/refund-notifications-job/refund-notifications-job.yaml (hunk 1 above)
Comment (larslunde-hmcts) [reply k8s/namespaces/fees-pay/refund-notifications-job/refund-notifications-job.yaml:17]: done

**Review by thomast1906 — DISMISSED** [review]

**Review by msl8r — DISMISSED** [review]

**Review by anshikanigam15 — APPROVED** [review]

**Review by anshikanigam15 — APPROVED** [review]

**Review by madjava — APPROVED** [review]

**Comment by larslunde-hmcts** [comment]
closing - was merged in with different PR

**Pull request closed without merging**